consensus/parlia/parlia.go

```diff
@@ -54,10 +54,12 @@ const (
 
 	checkpointInterval = 1024        // Number of blocks after which to save the snapshot to the database
 	defaultEpochLength = uint64(100) // Default number of blocks of checkpoint to update validatorSet from contract
+	defaultTurnTerm    = uint64(1)   // Default consecutive number of blocks a validator receives priority for block production
```

Comment:
can be uint8 directly

Reply:
accepted
the price is to add conversion when calculated with other variables